src/transaction.js

```diff
@@ -204,7 +204,7 @@ function deserializeTx (hexStr: string, withPrevScriptPubKey: boolean = false):
   var txObj = { version: 0, locktime: 0, ins: [], outs: [] }
 
   // Version
-  txObj.version = buf.readUInt32LE(offset)
+  txObj.version = buf.readInt32LE(offset)
```

Comment:
It should be good, like in bitcore-lib-zen (https://github.com/HorizenOfficial/bitcore-lib-zen/blob/zendoo-supportPR/lib/transaction/transaction.js#L376) but maybe test it.

Reply:
@MarcoOl94 we noticed that `readUInt32LE` breaks when we see negative version numbers (i.e. version: -4).

Do you see a better way to approach this issue?

Reply:
Which test did you do? Can you please try to deserialize a raw Tx, serialize the result and verify that the output is equal to the initial rawTx? (both with readUInt32LE and readInt32LE)

Reply:
Ok I'll approve it. Well done!